constructive_heuristics/elshafei_constructive.py

```diff
@@ -104,14 +104,11 @@ def elshafei_constructive(distance, flow):
     n = len(distance)
     solution = np.full((1,n), -1)[0]
     facility_mask, location_mask = np.zeros(n), np.zeros(n)
-
     location_rank, facility_rank = step1(n, distance), step2(n, distance, flow)
-
     # step 3: check whether any facility remains unassigned
     while (-1 in solution):
-        unique, counts = np.unique(solution, return_counts=True)
-        print(dict(zip(unique, counts))[-1])
-
+        #unique, counts = np.unique(solution, return_counts=True)
```

Comment:
can we remove this commented code? @harryxi1

Reply:
yes